pr_agent/tools/pr_reviewer.py

```diff
@@ -160,19 +162,17 @@ def _prepare_pr_review(self) -> str:
         Prepare the PR review by processing the AI prediction and generating a markdown-formatted text that summarizes
         the feedback.
         """
-        review = self.prediction.strip()
-
-        try:
-            data = json.loads(review)
-        except json.decoder.JSONDecodeError:
-            data = try_fix_json(review)
+        data = load_yaml(self.prediction.strip())
 
         # Move 'Security concerns' key to 'PR Analysis' section for better display
         pr_feedback = data.get('PR Feedback', {})
         security_concerns = pr_feedback.get('Security concerns')
-        if security_concerns:
+        if security_concerns is not None:
             del pr_feedback['Security concerns']
-            data.setdefault('PR Analysis', {})['Security concerns'] = security_concerns
+            if type(security_concerns) == bool and security_concerns == False:
+                data.setdefault('PR Analysis', {})['Security concerns'] = 'No security concerns found'
+            else:
+                data.setdefault('PR Analysis', {})['Security concerns'] = security_concerns
```

Comment:
**Suggestion:** Instead of checking if `security_concerns` is a boolean and equals False, you can simplify the condition by using `if not security_concerns:`. This will check for both `None` and `False` values.
```suggestion
        if security_concerns is not None:
            del pr_feedback['Security concerns']
            if not security_concerns:
                data.setdefault('PR Analysis', {})['Security concerns'] = 'No security concerns found'
            else:
                data.setdefault('PR Analysis', {})['Security concerns'] = security_concerns
```